testdata/cases/index.mjs

```diff
@@ -1,10 +1,13 @@
 import "./modules/options.js"
 import scenario from "./modules/scenario.js"
 
-// TODO do we want a "convention" here? An object
-// that can be used to replace value into, would be a start…
-const config = {
-  env: ENV || "staging",
+// NOTE: No 'var' or 'const' for defines!!!
+// Inject defines here with '--define defines.env="prod"'
```

Comment:
```suggestion
// NOTE: `--define` works on global identifiers only! To make `defines` a global identifier, it may not be defined via `var`/`let`/`const`.
// To replace fields of `defines`, use '--define defines.env="prod"'
```

Reply:
I used MUST NOT, otherwise applied.